src/py_sonar_scanner/scanner.py

```diff
@@ -29,11 +30,16 @@ class Scanner:
 
     def __init__(self, cfg: Configuration):
         self.cfg = cfg
+        self.log = logging.getLogger(__name__)
+        self.log.setLevel(logging.INFO)
+        handler = logging.StreamHandler()
+        handler.terminator = ""
+        self.log.addHandler(handler)
```

Comment:
It seems to me that there are a lot of lines involved in just setting up the logger. Would it perhaps make sense to extract it into its own function?